framework/PageActions/PageAction.tsx

```diff
@@ -51,12 +51,12 @@ interface IPageActionNoneCommon extends IPageActionCommon {
 interface IPageActionSingleCommon<T extends object> extends IPageActionCommon {
   selection: PageActionSelection.Single;
   isHidden?: (item: T) => boolean;
-  isDisabled?: string | ((item: T) => string | undefined);
+  isDisabled?: string | ((item: T) => string | undefined | boolean);
```

Comment:
these are causing the first 2 tsc failures... I don't think you want to add boolean here, you need to fix your isDisabled to return string or undefined

I think the convention is that isDisabled returns a string reason for why it's disabled, or undefined when not disabled; not just a boolean true/false

Reply:
Thank you, so helpful... do you mean that it's literally a plain-text description? I'll look at some other examples:)

Reply:
Probably something like this: `            : t`You do not have rights to this operation`,
`

Reply:
Yup, something like that, wrapped in a `t(...)` for l10n.